user-app-test/src/modules/users/entities/user.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { Column, Entity, PrimaryGeneratedColumn, Unique } from 'typeorm';
+
+@Entity()
```

Comment:
когда сделаешь миграции, то имеет смысл указать таблицу вручную, потому что тайпорм особо не заморачивается названиями
```suggestion
@Entity('users')
```